modules/installation-creating-aws-worker.adoc

```diff
@@ -0,0 +1,166 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-worker-{context}"]
+= Creating the worker nodes in AWS
+
+If you do not plan to automatically create worker nodes by using a MachineSet,
+create worker nodes in Amazon Web Services (AWS) for your cluster to use.
+The easiest way to manually create these nodes is to modify the provided
+CloudFormation template.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your worker
+nodes, you must review the provided information and manually create
+the infrastructure. You might need to consult Red Hat support to ensure that
+your cluster initializes correctly.
+====
+
+[WARNING]
+====
+It is better to use a MachineSet to allow the cluster to create and manage
```

Comment:
It is better to have a link to that section how to use machineset to provision new worker node.

Reply:
I'm going to create a new module based on https://docs.openshift.com/container-platform/4.0/machine_management/creating-machineset.html to include or link to.